app/src/main/java/com/hedvig/app/feature/insurance/ui/InsuranceAdapter.kt

```diff
@@ -280,23 +339,31 @@ class DashboardAdapter(
     }
 }
 
-sealed class DashboardModel {
-    object Header : DashboardModel()
+sealed class InsuranceModel {
+    object Header : InsuranceModel()
+
+    data class Renewal(
+        val renewalDate: LocalDate,
+        val draftCertificateUrl: String
+    ) : InsuranceModel()
+
 
     data class Contract(
-        val inner: DashboardQuery.Contract
-    ) : DashboardModel()
+        val inner: InsuranceQuery.Contract
+    ) : InsuranceModel()
 
     data class Upsell(
         @get:StringRes val title: Int,
         @get:StringRes val description: Int,
         @get:StringRes val ctaText: Int
-    ) : DashboardModel()
+    ) : InsuranceModel()
+
+    object Error : InsuranceModel()
 }
 
-class DashboardDiffUtilCallback(
-    private val old: List<DashboardModel>,
-    private val new: List<DashboardModel>
+class InsuranceDiffUtilCallback(
```

Comment:
I think this one is probably an exact match with `GenericDiffUtilCallback`, we can replace it while we're at it!